recipes/clingo/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = 'clingo' %}
+{% set version = "5.4.0" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/potassco/clingo/archive/v{{ version }}.tar.gz
+
+build:
+  number: {{ build }}
+  skip: True  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
```suggestion
    - {{ compiler('cxx') }}
    - make  # [unix]
```

Reply:
I can add it. But apparently there was one in the system anyway.

Reply:
There is on macOS and x86_64 but the conda-forge POWER images (disabled by default) don't contain `make` and it's generally cleaner to get it from conda-forge instead of relying on a quirk of the container.

Reply:
Okay.